Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "4.0.27"
+clap = { version = "4.0.27", features = ["derive"] }
```

Comment:
In the commit message explain why you did this.

Reply:
To use derive macros over `Args` struct. Message to remind myself.